R/estimate_infections.R

```diff
@@ -695,20 +695,16 @@ format_fit <- function(posterior_samples, horizon, shift, burn_in, start_date,
     format_out$samples <- format_out$samples[, strat := NA]
   }
   # add type based on horizon
-  # nolint start
   format_out$samples <- format_out$samples[
     ,
-    type := data.table::fifelse(
-      date > (max(date, na.rm = TRUE) - horizon),
-      "forecast",
-      data.table::fifelse(
-        date > (max(date, na.rm = TRUE) - horizon - shift),
-        "estimate based on partial data",
-        "estimate"
+    type := data.table::fcase(
+      date > (max(date, na.rm = TRUE) - horizon), "forecast",
+      date > (max(date, na.rm = TRUE) - horizon - shift),
+      "estimate based on partial data",
+      is.na(date), NA_character_,
+      default = "estimate"
       )
-    )
-  ]
-  # nolint end
+    ]
```

Comment:
This doesn't look like the right formatting

Reply:
I have tried addressing it here e8836198bdd9a0882cc7a8d7a24065c5141410cb. Do let me know if it's fixed.